docs/en/operations/utilities/clickhouse-disks.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /en/operations/utilities/clickhouse-disks
+sidebar_position: 59
+sidebar_label: clickhouse-disks
+---
+
+# clickhouse-disks
+
+A utility providing filesystem-like operations for ClickHouse disks.
+
+Program-wide options:
+
+* `--config-file, -C` -- path to ClickHouse config, defaults to `/etc/clickhouse-server/config.xml`.
+* `--save-logs` -- Log progress of invoked commands to `/var/log/clickhouse-server/clickhouse-disks.log`.
+* `--log-level` -- What [type](../server-configuration-parameters/settings#server_configuration_parameters-logger) of events to log, defaults to `none`.
+* `--disk` -- what disk to use for `mkdir, move, read, write, remove` commands. Defaults to `default`.
+
+## Commands
+
+* `copy [--diskFrom d1] [--diskTo d2] <FROM_PATH> <TO_PATH>`.
```

Comment:
It's strange to have command-line arguments in camelCase. Maybe `--disk-to`?

Reply:
Ok, let's change it